src/delete-activity-log.ts

```diff
@@ -137,17 +133,16 @@ export const handler = async (event: SNSEvent): Promise<void> => {
         console.log(
           `finished processing message with ID: ${record.Sns.MessageId}`
         );
-      } catch (err) {
-        console.error("Error in handler", err);
-        const message: SendMessageRequest = {
-          QueueUrl: DLQ_URL,
-          MessageBody: record.Sns.Message,
-        };
-        const result = await sqsClient.send(new SendMessageCommand(message));
-        console.error(
-          `[Message sent to DLQ] with message id = ${result.MessageId}`,
-          err
-        );
+      } catch (error) {
+        console.error(`[Error occurred]: ${(error as Error).message}`);
```

Comment:
Remind me - is there any risk `error.message` could contain part of the message payload? Our messages may contain personal data which we can't store in Cloudwatch.

Reply:
I believe it could be as we didn't log anything before, but removing would also make it difficult to figure out why our lambda failed... any suggestions? @alex9smith

Reply:
We need to be 100% confident we're not logging personal data, so I think in this case we have to accept that our logs may not be as useful. An error message like "Could not process message ${messageID}" might be the best we can do - that'll at least allow us to trace the logs back to a message and replay it.

We can always add debug logging to a lower environment to diagnose an issue if we need to.